webpack.config.js

```diff
@@ -1,20 +1,19 @@
-var webpack = require('webpack');
-var path = require('path');
-var NODE_ENV = process.env.NODE_ENV === 'Development';
+const webpack = require('webpack');
+const path = require('path');
+const isDev = process.env.isDev === 'Development';
```

Comment:
Usually developers use NODE_ENV variable to define environment. It will be not obvious for new developers that environment defined in process.env.isDev
isDev variable inside config - it`s ok